docs/user/ppl/cmd/lookup.rst

```diff
@@ -0,0 +1,139 @@
+    | 13               | Alice    |
+    +------------------+----------+
+
+    os> source=accounts | lookup hr account_number;
+    fetched rows / total rows = 2/2
+    +------------------+----------+----------+
+    | account_number   | gender   | name     |
+    |------------------+----------|----------|
+    | 1                | M        | John     |
+    | 13               | F        | Alice    |
+    +------------------+----------+----------+
+
+
+Example 2: Lookup with different field names
+============================================
+
+The example show a lookup to add the name of a person from a lookup index with different field names.
+
+PPL query::
+
+    os> source=accounts;
+    fetched rows / total rows = 2/2
+    +------------------+----------+
+    | account_number   | gender   |
+    |------------------+----------|
+    | 1                | M        |
+    | 13               | F        |
+    +------------------+----------+
+
+    os> source=hr;
+    fetched rows / total rows = 2/2
+    +------------------+----------+
+    | employee_number  | name     |
+    |------------------+----------|
+    | 1                | John     |
+    | 13               | Alice    |
+    +------------------+----------+
+
+    os> source=accounts | lookup hr employee_number AS account_number name AS given_name;
+    fetched rows / total rows = 2/2
+    +------------------+----------+----------------+
+    | account_number   | gender   | given_name     |
+    |------------------+----------|----------------|
+    | 1                | M        | John           |
+    | 13               | F        | Alice          |
+    +------------------+----------+----------------+
+
+Example 3: Full lookup example
+==============================
+
+The example show a lookup to add the name of a person from a lookup index with different field names.
+
+PPL query::
+
+    os> source=accounts;
+    fetched rows / total rows = 3/3
+    +------------------+----------+------------+
+    | account_number   | gender   | department |
+    |------------------+----------+------------|
+    | 1                | M        | finance    |
+    | 13               | F        | it         |
+    | 20               | M        | it         |
+    +------------------+----------+------------+
+
+    os> source=hr;
+    fetched rows / total rows = 4/4
+    +------------------+----------+------------+--------+
+    | employee_number  | name     | dep        | active |
+    |------------------+----------|------------|--------|
+    | 1                | John     | finance    | true   |
+    | 13               | Alice    | finance    | false  |
+    | 13               | Melinda  | it         | true   |
+    | 19               | Jack     | finance    | true   |
+    +------------------+----------+------------+--------+
+
+    os> source=accounts | lookup hr employee_number AS account_number, dep AS department appendonly=true name AS given_name, active AS is_active ;
```

Comment:
Can you please also add same example with appendonly=false ? I think that will help clarify how appendonly works. `source=accounts | lookup hr employee_number AS account_number, dep AS department appendonly=false name AS given_name, active AS is_active ;`

Reply:
Will add it

Reply:
done